app/proc/twitter_test.go

```diff
@@ -4,9 +4,64 @@ import (
 	"strconv"
 	"testing"
 
+	"github.com/umputun/feed-master/app/feed"
+
 	"github.com/stretchr/testify/assert"
 )
 
+func TestNewTwitterClient(t *testing.T) {
+	twiAuth := TwitterAuth{
+		ConsumerKey:    "a",
+		ConsumerSecret: "b",
+		AccessToken:    "c",
+		AccessSecret:   "d",
+	}
+
+	twitterFmtFn := func(item feed.Item) string {
+		return ""
+	}
+
+	client := NewTwitterClient(twiAuth, twitterFmtFn)
+
+	assert.Equal(t, client.TwitterAuth.ConsumerKey, "a")
```

Comment:
Probably could be replaced with checking complete structure instead of checking per-property.

Reply:
Done